metadata.txt

```diff
@@ -11,7 +11,7 @@
 [general]
 name=OpenQuake Integrated Risk Modelling Toolkit
 qgisMinimumVersion=2.0
-description=<p>Tools for creating and editing indicators and composite indices and for combining these with estimates of physical earthquake risk (i.e. estimates of human or infrastructure loss).</p><p>The plugin enables users to directly interact with the <a href='https://platform.openquake.org'>OpenQuake Platform</a>, in order to browse and download socioeconomic data or existing projects, to edit projects locally in QGIS, then to upload and share them through the Platform.</p><p>This plugin has been created by the <a href='http://www.globalquakemodel.org'>GEM Foundation</a> and was formerly known as GEM OpenQuake Social Vulnerability and Integrated Risk (SVIR)</p>
+description=<p>Tools for creating and editing indicators and composite indices and for combining these with estimates of physical earthquake risk (i.e. estimates of human or infrastructure loss).</p><p>The plugin enables users to directly interact with the <a href='https://platform.openquake.org'>OpenQuake Platform</a>, in order to browse and download socioeconomic data or existing projects, to edit projects locally in QGIS, then to upload and share them through the Platform.</p><p>This plugin was designed as a collaborative effort between the <a href='http://www.globalquakemodel.org'>GEM Foundation</a> and the <a href='http://www.cedim.de/english/'>Center for Disaster Management and Risk Reduction Technology</a>, and it has been developed by the GEM Foundation. It was formerly named GEM OpenQuake Social Vulnerability and Integrated Risk (SVIR)</p>
```

Comment:
You may consider updating the description just slightly to offer more information regarding the type of indices most commonly developed within the tool. One option for the new description could be, ”Tools for creating and editing indicators and composite indices to measure social characteristics and for combining these with estimates of physical risk (i.e. estimates of human or infrastructure loss).” We're just adding "to measure social characteristics".